pkg/apis/addtoscheme_sro_v1alpha1.go

```diff
@@ -0,0 +1,10 @@
+package apis
+
+import (
+	"github.com/zvonkok/special-resource-operator/pkg/apis/sro/v1alpha1"
```

Comment:
need to reference its own base github.com/nvidia/gpu-operator

Reply:
Here as well all the references to SRO should be changed to nvidia.